recipes/callerpp/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "callerpp" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/nh13/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 647cec22189639bb35ea84ce682edb159496e78ad712aa966dcea2e53f6da163
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - spoa >=4.1.4
+
+test:
+  commands:
+    - callerpp --version
+    - printf ">1\nAAAAAGAAAAA\nAAAAACAAAAA\nAAAAAGAAAAA" > ref.fa; callerpp -a 2 --input ref.fa 
+
+about:
+  home: https://github.com/nh13/callerpp
+  license: MIT
+  license_file: LICENSE
+  summary: 'A simple consensus caller based on partial order alignment with spoa'
+
+extra:
```

```suggestion
extra:
  additional-platforms:
    - linux-aarch64
```
consider adding support for Linux ARM64 !